DESCRIPTION

```diff
@@ -35,8 +35,7 @@ URL: https://igraph.org, https://igraph.discourse.group/
 SystemRequirements:
     gmp (optional),
     libxml2 (optional),
-    glpk (>= 4.57, optional),
-    C++17
```

Comment:
@krlmlr This change breaks the CRAN pretests again with the following `NOTE`:

```
* checking C++ specification ... NOTE
  Specified C++11: please update to current default of C++17
```

I'm going to revert this for the sake of the submission and then we should decide what the long-term solution should be.

Reply:
Where did we specify C++11?

Reply:
Oh, got it, in `CXX_STD` .